java/arcs/sdk/wasm/WasmCollectionImpl.kt

```diff
@@ -85,4 +85,6 @@ class WasmCollectionImpl<T : WasmEntity>(
             action(s)
         }
     }
+
+    override fun getEntity(): Set<T> = fetchAll()
```

Comment:
This seems a weird method name for a Collection. `getContent` might be slightly better? `fetch`?

Another way would be to just switch on the type of the handle in the calling code. Which defeats the point of introducing the `WasmHandleEvents` class in the first place. But you could just fold the `WasmHandleEvents` code into the `WasmHandle` class.

Reply:
The issue I was trying to resolve here is that Singleton has fetch() and Collection has fetchAll(), but I need them to have the same name. I like getContent(). I didn't put this in the Wasm Handle.kt to better match the PA code. I could change both of them if we think that would be helpful?

Reply:
I think it's OK if they're different (for now)